packages/babel-preset-app/README.md

```diff
@@ -69,14 +67,16 @@ There are [detailed docs](https://babeljs.io/docs/en/babel-preset-env#options) f
 export default {
   build: {
     babel: {
-      presets({ isServer }) {
+      presets({ envName }) {
+        const envTargets = {
```

Comment:
@clarkdo should `client` and `server` be swapped?

Reply:
Correct, thanks.